apiserver/common/mocks/machine_mock.go

```diff
@@ -0,0 +1,80 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: github.com/juju/juju/apiserver/common (interfaces: MachineService)
+//
+// Generated by this command:
+//
+//	mockgen -typed -package mocks -destination mocks/machine_mock.go github.com/juju/juju/apiserver/common MachineService
+//
+
+// Package mocks is a generated GoMock package.
+package mocks
```

Comment:
I dislike separate mocks packages, but I see there's a precedent for the cases you've added them here.